onset_maproom/calc.py

```diff
@@ -27,11 +27,12 @@ def onset_date(
     time_coord="T",
 ):
     """Function reproducing Ingrid onsetDate function
-    http://iridl.ldeo.columbia.edu/dochelp/Documentation/details/index.html?func=onsetDate
+    http://iridl.ldeo.columbia.edu/dochelp/Documentation/details/index.html
```

Comment:
Changing a URL to keep it from going over 88 characters is probably being overzealous.

Reply:
I don't know what is at stake here...

Reply:
Just readability. My editor window wasn't wide enough to fit your long docstring lines, so they wrapped around, which disturbs the appearance of the indentation, which makes it harder for me to see the shape of the code blocks. (I assume it's obvious what's at stake when you change the URL.)

Reply:
Put back the original URL.

http://iridl.ldeo.columbia.edu/dochelp/Documentation/details/index.html?func=onsetDate